custom_components/frigate/__init__.py

```diff
@@ -26,7 +26,8 @@
 from homeassistant.components.switch import DOMAIN as SWITCH_DOMAIN
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import ATTR_MODEL, CONF_HOST, CONF_URL
-from homeassistant.core import Config, HomeAssistant, callback, valid_entity_id
+from homeassistant.core import HomeAssistant, callback, valid_entity_id
+from homeassistant.core_config import Config
```

Comment:
If this is only used for typing the parameter in `async_setup`, we can probably remove it. That parameter isn't an instance of `Config`.

Reply:
So add
`from homeassistant.helpers.typing import ConfigType`
and change line 176 to
`async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:`
?

Reply:
to make the change work currently we would need to update the required home assistant version. I think it would be best to just remove the usage of `Config` since it was only used as a type